.github/workflows/CI.yaml

```diff
@@ -81,6 +92,7 @@ jobs:
       fail-fast: false
       matrix:
         project: ['common', 'core', 'processor', 'test-utils', 'it']
+        java-version: ['17', '21']
```

Comment:
I remember there was a reason we had 17.0.8, but not what that reason was, is it no longer a problem?

Reply:
Because we had to move it up specifically with a breaking change they did in JDK? But that shouldn't happen hopefully very often